simple/src/codingPractice/DivideString.java

```diff
@@ -2,16 +2,16 @@
 
 public class DivideString {
     public static void main(String[] args) {
-        String stringComplete = "aaaabbbbcccc";
-        int length = stringComplete.length();
+        String bunchOfString = "aaaabbbbcccc";
```

Comment:
Use proper variable value instead of "aaaabbbbcccc".

Reply:
done